var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -31,6 +31,9 @@ class Steps(CMakePackage):
     variant("bundle", default=False, description="Use bundled libraries")
     variant("stochtests", default=True, description="Add stochastic tests to ctests")
     variant("timemory", default=False, description="Add timemory API to instrument time/memory")
+    variant("build_type", default="Release", description="CMake build type",
```

Comment:
Keep previous spack default
```suggestion
    variant("build_type", default="RelWithDebInfo", description="CMake build type",
```